cli/pcluster/config/mappings.py

```diff
@@ -51,20 +54,22 @@
     efa_validator,
     efs_id_validator,
     efs_validator,
+    fsx_arch_support,
```

Comment:
The reason I did it like this is because there already existed a validator named `fsx_os_support`. I agree that it's better to have all validator names end in `_validator`, so I'm going to change both.